codice/ddf
# DDF-4759 updated script to automate product site

#### What does this PR do?

replaces manual updating of product website `index`, `download`, and `documentation-versions` pages with scripting.

#### Select relevant component teams: 
@codice/docs 
@codice/website 

#### Ask 2 committers to review/merge the PR and tag them here.
@ahoffer
@brjeter
@clockard
@djblue
@jrnorth
@lessarderic
@ricklarsen - Documentation
@shaundmorris

#### How should this be tested?
load site into browser. 
verify content and formatting on these pages
- index
- download
- documentation-version

#### Any background context you want to provide?
This script will not update the documentation artifacts themselves.
Some older versions will remain hard-coded as they were maintained in a different repository 

#### What are the relevant tickets?
#4759 

#### Screenshots

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
